test/complex.jl

```diff
@@ -242,8 +241,7 @@ end
 end
 
 @testset "exp" begin
-    #  exp(conj(z)) = conj(exp(z))
-
+    # exp(conj(z)) = conj(exp(z))
```

Comment:
Likewise here, perhaps incorporate the comment in the testset description and nix the comment?